app/src/components/pages/collectionPage/collectionPage.tsx

```diff
@@ -140,51 +179,79 @@ const CollectionPage = ({
                 alignItems: "flex-start",
               }}
             >
-              <Heading
-                size="heading5"
-                ref={headingRef}
-                tabIndex={-1}
-                margin="0"
-                aria-live="polite"
-              >{`Displaying ${displayResults(
-                searchResults.numResults,
-                CARDS_PER_PAGE,
-                1
-              )} results`}</Heading>
-              <SortMenu
-                options={SEARCH_SORT_LABELS}
-                searchManager={collectionSearchManager}
-                updateURL={updateURL}
-                setFiltersExpanded={setFiltersExpanded}
-              />
+              {searchResults.results.length > 0 && (
+                <>
+                  <Heading
```

Comment:
This is also shared with `searchPage` and _could_ be a small component just for layout and consistent DOM purposes. It might be tedious to have a single component with props for both internal `Heading` and `SortMenu` components.